index.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  extends: ['stylelint-config-standard-scss'],
+  rules: {
+    // forces the use of min-width (mobile-first) based styling
+    'media-feature-name-disallowed-list': ['max-width'],
```

Comment:
@ajobi Is this really necessary? Seems limitating. I mean i totally agree on using a mobile first approach, but couldn't this option be useful in some cases?

Reply:
@dz000 this one is actually the less limiting one compared to type selectors. Any query using `max-width` can always be written with `min-width` instead right?

If you have a special case where the linting is limiting you can disable it for that local case and still keep the warnings elsewhere:
`
// stylelint-disable-next-line selector-max-id
#externally-inserted-element-id {
  display: none;
}
`